src/TestProblem/ELBDM/LSS/Init_TestProb_ELBDM_LSS.cpp

```diff
@@ -4,7 +4,14 @@
 
 // problem-specific global variables
 // =======================================================================================
-static int LSS_InitMode;   // initialization mode: 1=density-only, 2=real and imaginary parts or phase and density
+static int  LSS_InitMode;   // initialization mode: 1=density-only, 2=real and imaginary parts or phase and density
+static double ZoomIn_Center_x;  // x coordinate of refinement region center
+static double ZoomIn_Center_y;  // y coordinate of refinement region center
+static double ZoomIn_Center_z;  // z coordinate of refinement region center
```

Comment:
[Optimize] I think the `ZoomIn_Center_*` can be as a function of `a` which is the most general case.

---

If you agree, please let `ZoomIn_Center_*` be a part of `Input__ZoominBox`. Also, please add a check of the zoom-in region (the next zoom-in region can only lie in the previous region).

Reply:
it is already in `Input__TestProb`